about.html

```diff
@@ -0,0 +1,101 @@
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque quisquam cum fugit nulla rerum aliquam quia commodi debitis obcaecati sequi ut vero, aut rem sint totam dolorum laborum accusantium suscipit! </p>
+      </div>
+      <p class="p-email">Please contact us via email for further questions <br> about AR Spark Training 2022
+      <br> <a id="home-mail" href="kamzzytech@gmail.com">sparktraining@kamzzylearns.org</a>
+    </p>
+       <div class="about2">
+         <h4>AUGUMENTED REALITY</h4>
+         <h4>SPARK TRAINING 2022 Concept</h4>
+       <hr class="hr">
+       <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis, voluptatibus?</p>
+     </div>
+     <div class="pastT">
+       <h4>View previous AR Spark Trainings</h4>
+       <hr class="hr">
+       <p>Consider the previous two AR Spark Trainings, which were held in Lagos and Abuja.</p>
+      <div class="row g-1">
+        <div class=" col-sm-12 col-md-5 card1">
+        </div>
+        <div class="col-sm-12 col-md-5 card2">
+        </div>
+      </div>
+       
+     </div>
+       <footer>
+        <h4>Partners</h4>
+        <hr class="hr">
+        <div class="row">
+          <div class="col-6 col-sm-4 col-md-2">
+           <a href="https://kamzzy-url-shortner.herokuapp.com/"> <img src="https://res.cloudinary.com/wonderwoman/image/upload/v1643937096/myportfolio/url-shortener_logo_erqj1q.png" alt="partnerLogo">
+         </a>
+          </div>
+          <div class="col-6 col-sm-4 col-md-2">
+           <a href="https://kamzzywallet.herokuapp.com/"> <img src="https://res.cloudinary.com/wonderwoman/image/upload/v1650469130/myportfolio/Pink_Elegant_Lash_Makeup_Brow_artist_Beauty_Logo_2_sv8dqp-removebg-preview_cpguin.png" alt="partnerLogo">
+         </a>
+          </div>
+          <div class="col-6 col-sm-4 col-md-2">
+            <a href="https://fervent-bell-7c4004.netlify.app/"><img src="https://res.cloudinary.com/wonderwoman/image/upload/v1643601874/myportfolio/foodie_logo_hhevph.png" alt="partnerLogo">
+         </a>
+            </div>
+          <div class="col-6 col-sm-4 col-md-2">
+            <a href="https://elastic-neumann-4b0e32.netlify.app/"><img src="https://res.cloudinary.com/wonderwoman/image/upload/v1650468927/myportfolio/KAMZZY_opqskv-removebg-preview_brsixs.png" alt="partnerLogo">
+         </a>
+            </div>
+          <div class="col-6 col-sm-4 col-md-2">
+            <a href="https://brave-wright-719995.netlify.app/">
+              <img src="https://res.cloudinary.com/wonderwoman/image/upload/v1597587730/myportfolio/calculator_twqynb.png" alt="partnerLogo"></a>
+          </div>
+        </div>
+        <div class="endDiv row">
+    <div class="col-6">
+      <a class="logo navbar-brand" href="/index.html"> <img src="/icons/kamzzylearns.png" alt="logo"> </a>
+    </div>
+    <div class="col-6">
+    <p class="fEnd"> 2022 Kamzzy Learns Training <br> All  Rights Reserved</p>
+    </div>
+        </div>
```

Comment:
- [ ] Your partner's section on both the 
![Captura de pantalla 2022-04-21 002741](https://user-images.githubusercontent.com/59100477/164335231-afceb295-ed57-4a26-9698-ac617f6c8030.jpg)
about and home page are not being aligned as expected. They seem to be disorganized, so would be nice if you can give them a proper alignment to the center or the one you consider best. Here's an image to relate: